tests/bluetooth/tester/src/btp/btp_bap.h

```diff
@@ -43,3 +51,12 @@ struct btp_ascs_ase_found_ev {
 	uint8_t dir;
 	uint8_t ase_id;
 } __packed;
+
+#define BTP_BAP_EV_STREAM_RECEIVED	0x83
+struct btp_bap_stream_received_ev {
+	bt_addr_le_t address;
+	uint8_t dir;
```

Comment:
Why is `dir` relevant? For the client, it will always be `SOURCE` and for the server it will always be `SINK` right?